dashboard/app/dsl/script_dsl.rb

```diff
@@ -68,6 +70,7 @@ def level(name, properties = {})
     @levels << {
       :name => name,
       :stage => @stage,
+      :stage_flex_category => @stage_flex_category,
```

Comment:
As a tangent, eventually I'd like to unlink `Script` and `ScriptLevel` and always go through `Stage` (which is a little weird because `ScriptLevel` was originally a `has_many :through` join table for `Script` and `Level`).

Now that all scripts have stages, I think a strict hierarchy makes sense between `Script` -> `Stage` -> `ScriptLevel` (maybe rename `ScriptLevel` -> `Puzzle` to match the URL structure?).